gobblin-compaction/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This file is copied from gobblin-runtime module. User can specify [-i] option in ./gradlew to see the map-reduce console log when running compaction tests. I have remove this file now.